packages/contracts/test/issues/CHU663/CHU663.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../../lib/forge-std/src/Script.sol";
+import "../../../lib/forge-std/src/Test.sol";
+import { Sphinx } from "../../../contracts/foundry/Sphinx.sol";
+
+contract CHU663_test is Test, Sphinx {
+    address public safe;
+
+    function configureSphinx() public override {
+        sphinxConfig.projectName = "CHU-663";
+        sphinxConfig.owners = [0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266];
+        sphinxConfig.threshold = 1;
+        sphinxConfig.orgId = "test-org-id";
+        safe = safeAddress();
+    }
+
+    function test_configureSphinx_success_calls_safeAddress() external {
+        configureSphinx();
+        assertNotEq(safe, address(0));
+    }
+}
```

Comment:
Generally, I think our current test suite is quite unmanageable as it is largely due to the fact that we're continuously adding more tests to monolithic files, which make it difficult to look for any particular thing. I don't think the current pattern is maintainable in the long term because we will end up with ridiculously huge test files for SphinxUtils.sol and Sphinx.sol. 

This test is an example of a pattern implemented in many other places that is significantly better in my opinion, but it's just one option. I'm happy to talk more about it on Friday.